pages/about.js

```diff
@@ -26,17 +28,19 @@ export default function About({ leads }) {
       <MissionVision />
       <Services servicesData={servicesData} />
       <Team leads={leads} />
+      <Founders founders={founders}></Founders>
       <JoinUs />
     </Layout>
   );
 }
 
 export async function getServerSideProps() {
   const leads = await getTeam();
```

Comment:
we can combine `getFounder` and `getTeam` into one function and pass a filter key: `Position` or a filter condition